### ISSUE TITLE
Blank screen - internal server error resulted in undefined being called

### ISSUE THREAD

**Issue by HappyBros** [elm-lang/elm-reactor#59]
I tried to debug this mouse tracker code:

    import Graphics.Element as Element
    import Graphics.Collage as Collage
    import Signal as Signal
    import Color as Color
    import Text
    import Mouse as Mouse
    
    
    makeScene : Float -> Collage.Form
    makeScene sideLength = Collage.outlined (Collage.solid Color.black) (Collage.square sideLength)
    
    renderScene : Collage.Form -> (Int, Int) -> Element.Element
    renderScene scene (x, y) = Collage.collage (x*2) (y*2) [scene]
    
    main : Signal.Signal Element.Element
    main = Signal.map (renderScene <| makeScene 10.0) Mouse.position

It worked when run stand-alone. However, while debugging, I got a blank screen on Chrome. As suggested [here](https://github.com/elm-lang/elm-reactor/issues/58), I opened the Developer Console to get the error, and I saw the following:
```
(X) [http://localhost:8000/debugging-runtime.js] Failed to load resource: the server responded with a status of 500 (Internal Server Error)
(X) [debugger.js:11900] Uncaught TypeError: undefined is not a function
```
I assume this is a bug within debugging-runtime.js that caused debugger.js to call `undefined`. And if it helps, I installed Elm Platform 0.14.

**Comment by evancz** [comment]
Thanks @lsjroberts! @HappyBros, is that a correct assessment?

You can follow [these instructions](https://groups.google.com/forum/#!topic/elm-discuss/KHg-aeFvDN0) to get the very latest version which should fix a bunch of issues.

**Comment by evancz** [comment]
I'm going to close as of the release of 0.14.1

**Comment by HappyBros** [comment]
Thanks :)

**Comment by evancz** [comment]
Cool, glad it's working!

**Comment by peacememories** [comment]
I know this issue has been closed for a while now, but my problem seems to be the same and so far I haven't found any solution.

On MacOS, with the latest `elm-platform` (0.18 if I am not mistaken), and a very simple `Dice.elm`:

```
module Dice exposing (..)

import Html exposing (Html, div, text)


main =
    div [] [ text "Hello World!" ]
```

About 90% of the time when I refresh the page `localhost:8000/Dice.elm` I am greeted with a blank page, and the console outputs the following:

```
[Error] Failed to load resource: the server responded with a status of 500 (Internal Server Error) (Dice.elm, line 0)
[Error] ReferenceError: Can't find variable: runElmProgram
	Global Code (Dice.elm:10)
```

The resource in question is `/_compile/Dice.elm` which returns an error code 500.

I have uninstalled and reinstalled elm-platform a few times, and originally I was using the elm binaries from Homebrew, with the same result.

There could be a problem with my setup, but if so I can't find it.

Sorry for unearthing an old issue and thank you for taking the time to read this :)
